observability/monitoring/reviewing-monitoring-dashboards.adoc

```diff
@@ -6,40 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-ifndef::openshift-dedicated,openshift-rosa[]
-{product-title} {product-version} provides a comprehensive set of monitoring dashboards that help you understand the state of cluster components and user-defined workloads.
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
-{product-title} provides monitoring dashboards that help you understand the state of user-defined projects.
-endif::openshift-dedicated,openshift-rosa[]
-
-Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following items:
-
-* API performance
-* etcd
-* Kubernetes compute resources
-* Kubernetes network resources
-* Prometheus
-* USE method dashboards relating to cluster and node performance
-* Node performance metrics
-
-.Example dashboard in the Administrator perspective
-image::monitoring-dashboard-administrator.png[]
-
-Use the *Developer* perspective to access Kubernetes compute resources dashboards that provide the following application metrics for a selected project:
-
-* CPU usage
-* Memory usage
-* Bandwidth information
-* Packet rate information
-
-.Example dashboard in the Developer perspective
-image::observe-dashboard-developer.png[]
-
-[NOTE]
-====
-In the *Developer* perspective, you can view dashboards for only one project at a time.
-====
+// About monitoring dashboards
+include::modules/monitoring-about-monitoring-dashboards.adoc[leveloffset=+1]
```

Comment:
The introduction is now included here

Reply:
Good call. Can you add a very short summary before the module so we don't get back-to-back headings?
![image](https://github.com/user-attachments/assets/202706db-1ba4-4bf0-9f27-66d0f1642019)